### DISCUSSION TITLE
The new VideoDownloadHelper for Firefox is buggy

### DISCUSSION THREAD

**Discussion by mickpf** [aclap-dev/video-downloadhelper#637] [Help and Q&A]
### Your question

Hello,

In my opinion, the new version 9.0.1.13 of VideoDownloadHelper for Firefox has some bugs.

1. You can activate the sidebar for the status of running downloads, which is very nice. The problem is that you can no longer deactivate it: every time you start a download, the sidebar is activated, even if you don't want it to be. How to disable the sidebar?
2. The settings for the addon are no longer accessible at all, as it was possible in the previous version 8.x.

Thanks in advance,
Michael

### Addon & CoApp version and details

Addon version: 9.0.1.13
CoApp version: 2.0.19.0

**Comment by Maccadoodle67** [comment]
https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/versions/

**Reply by GrampaWildWilly** [reply]
The Internet is changing, & VDH is doing its best to keep up.  If you stick on an old release, you will eventually find there are things you want to do that are not available in what you are using.  You will eventually need to get on the current release.  I stuck with Windows 7 for over 10 years, well past the out-of-service date announced by Microsoft.  But I eventually felt the antiquity of not only W7 but of my hardware.  I'm of the if it ain't broke, don't fix it school.  But I had to reluctantly admit, finally, it was not exactly broke but neither was it working right.  Progress is inevitable.  Your only choice is how soon you adapt to it.

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/722661c7-fe1f-4ad7-b186-15da4292a33c)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/2ff83f3b-2190-4e61-912f-0796386d16b7)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/657ec59c-22db-49da-98e0-61756462028a)
![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6a9e73d6-b936-4758-9931-ddb925de4f06)
![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0ae29f9f-1986-4e9b-8291-6b2a46a798d5)
![#06](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/24a6e82c-a084-443c-b649-51ffe10df45d)
![#07](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/3939b71b-3d36-489a-bb85-4149995a9b9f)
![#08](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0369eb74-adc0-4a4f-b028-d2a4a2461b56)
![#09](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e00970c4-12f5-4069-9d72-04c81dd11648)

**Comment by mickpf** [comment]
Many Thanks for help!

I didn't see the very small buttons (on my windows tablet) at the bottom of the sidebar... Sorry!

**Reply by 40ZqTYY4ag3tAO4hdlxR1** [reply]
+1 on that ... I was struggling with the same issue. Came here to look and thanks to the awesome post and screenshots by @GrampaWildWilly who has better 👓 eyesight 👓 than me.... finally figured this out after a few days of getting annoyed by the sidebar popping up. 🫡

**Comment by Maccadoodle67** [comment]
I'm gonna stick with V8

**Reply by GrampaWildWilly** [reply]
Suit yourself.  But don't report any problems with it.  You will be met with the response that we're not having any trouble with VDH 9 . . . or whatever will be current at the time.

**Reply by PlayaCharacter** [reply]
Someone is telling you they are having trouble with it right now and you're ignoring their feedback, so the utility of reporting problems is questionable regardless of what version they are using. I'll second their comment, I'm reverting to an older version because this new UI is obstructive, cumbersome, needlessly difficult to configure, it consumes valuable screen real estate and fills it with blank space for no apparent purpose, and appears to exist purely for its own sake. The old UI left much to be desired, but at least it was navigable and unobtrusive.

**Reply by GrampaWildWilly** [reply]
I stated clearly above in my images.  I do not work for VDH.  I am just another user.  Look carefully.  So I am not ignoring anything.  I can't fix anything, but I don't ignore anything.

If you simply must have the legacy user interface, there is a setting in the new interface for enabling that.  If you look at my images carefully, you'll see it.  If an image is too small for you, click it.  The way GitHub works, if you click an attached image, GitHub will display it in a new browser window.  Usually.  I have encountered times when that doesn't work.  That seems to be a transient problem with the GitHub web site.  Some time later, perhaps minutes, perhaps hours, something that failed once again works.  I can't explain that.  But clicking an image is supposed to open it in a separate browser window where you can use the browser's own magnifier feature to zoom in on something that is small.

Now, you must understand that the legacy UI is not going to exist forever.  Eventually, @paulrouget Paul is going to remove it.  There are still some features of past releases & settings in past releases that he has not ported forward to the new release.  But there will come a point when he'll be done & there will be no more use for the old interface.  You can at least get the benefits of the new version but continue to use the old UI if you really want.  If you actually stick on an old release, you are going to back yourself into a corner.  It is inevitable that you will hit something that flat doesn't work in the old release.  It may take you a week.  It may take you 2 years.  But at some point, you're going to hit a problem.  You're going to report it here.  And the response you're going to get will be, "It works in the new release.  Upgrade or quit complaining."  You know this is the way of the world.  Like I said above, progress is inevitable.  Your only choice is how soon you adapt to it.

**Reply by GrampaWildWilly** [reply]
As for the new interface taking up screen space, you need to turn off the sidebar.  Once again, I refer you to my images above.  Magnify them if necessary.  But there is a way to put the VDH menu back in its box as a menu instead of a sidebar.  You just have to avail yourself of the available control.  I don't use the sidebar myself.  I agree with you that it shouldn't take up valuable screen real estate.  That's the biggest reason I don't use the sidebar.  But it's available for those who like it.  You & I don't like it.  So I don't use it & you should do the same.

**Reply by GrampaWildWilly** [reply]
On the subject of the old UI being unobtrusive, I can take issue with that.  To open the settings in the old UI, VDH used to open an entire new browser tab.  A whole window.  In the new UI, all the settings are in the drop-down menu.  I say that's an improvement, less obtrusive than the old interface.  Also, the old interface forced you to remember to click the Save button.  There is no need for that technique for most things in the new UI.  Here & there, there is still a Save button.  But for the most part, changing a setting takes immediately & you don't have to remember to click Save.

**Reply by GrampaWildWilly** [reply]
Another improvement.  In the old UI, each variant was its own line on the VDH drop-down menu.  In the new UI, a video has one entry.  Just one.  The various resolutions are packed away in their own drop-down within the main menu.  This is another way the new UI is less intrusive than the old.
